### ISSUE TITLE
colors.webpage.prefers_color_scheme_dark breaks with Qt 5.15.2

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#5915]
```
.tox/py38-pyqt515/bin/python3 -m qutebrowser --temp-basedir -s colors.webpage.prefers_color_scheme_dark true ':later 500 jseval matchMedia("(prefers-color-scheme: dark)").matches'
```

This shows "True" on QtWebEngine 5.15.1 and below, but "False" on QtWebEngine 5.15.2.

I suspect the `--force-dark-mode` Chromium flag got removed/renamed between Chromium 80 and 83.

**Comment by The-Compiler** [comment]
According to `ekpneo` who originally reported this in IRC, passing `--blink-settings=preferredColorScheme=1` instead should work.

**Comment by travankor** [comment]
What's the full --qt-flag invocation now?

**Comment by The-Compiler** [comment]
@travankor `--qt-flag blink-settings=preferredColorScheme=1` (note you can also add it to the `qt.args` setting instead)

**Comment by travankor** [comment]
Oh wait, is `blink-settings=forceDarkModeEnabled=true` supposed to always override this setting? I guess there isn't a way to have `prefers_color_scheme_dark` work with `darkmode.enabled`.

**Comment by The-Compiler** [comment]
No idea, whatever the underlying Chromium/Blink does - setting `colors.webpage.darkmode.policy.page` to `smart` might be enough for it to realize that the page is already dark? In theory there also should be a way for websites to turn off the recoloring (see #5542) but I never got it to work properly.

**Comment by realSConway** [comment]
Hi guys,
Is this reason why darkmode isnt working for me atm?
```
qutebrowser v1.14.0
Backend: QtWebEngine (Chromium 83.0.4103.122)
Qt: 5.15.2

CPython: 3.8.6
PyQt: 5.15.1

sip: 5.4.0
colorama: no
pypeg2: 2.15
jinja2: 2.11.2
pygments: 2.7.2
yaml: 5.3.1
cssutils: no
attr: 20.2.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.1
PyQt5.QtWebKitWidgets: yes
pdf.js: no
sqlite: 3.33.0
QtNetwork SSL: OpenSSL 1.1.1h  22 Sep 2020

Style: QGtkStyle
Platform plugin: xcb
OpenGL: X.Org, 4.6 (Compatibility Profile) Mesa 20.2.3
Platform: Linux-5.9.10-1-default-x86_64-with-glibc2.2.5, 64bit
Linux distribution: openSUSE Tumbleweed (unknown)
Frozen: False
Imported from /usr/lib/python3.8/site-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib64/qt5/libexec, data path: /usr/share/qt5
OS Version: 

--- /etc/os-release ---
NAME="openSUSE Tumbleweed"
# VERSION="20201127"
ID="opensuse-tumbleweed"
ID_LIKE="opensuse suse"
VERSION_ID="20201127"
PRETTY_NAME="openSUSE Tumbleweed"
CPE_NAME="cpe:/o:opensuse:tumbleweed:20201127"
DOCUMENTATION_URL="https://en.opensuse.org/Portal:Tumbleweed"
LOGO="distributor-logo"

Paths:
cache: $HOME/.cache/qutebrowser
config: $HOME/.config/qutebrowser
data: $HOME/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser

Autoconfig loaded: no
Config.py: $HOME/.config/qutebrowser/config.py has been loaded
Uptime: 3:37:13`
```

If I can help with anything let me know.

**Comment by The-Compiler** [comment]
@realSConway No, in your case the reason is that you're on Qt 5.15.2 but PyQt 5.15.1. Either upgrade PyQt, or use qutebrowser from git master and `export QUTE_DARKMODE_VARIANT=qt_515_2` as a workaround.

**Comment by realSConway** [comment]
Thanks for the reply.

I've upgraded my opensuse snapshot to 20201105 thinking this may resolve it. Which it didn't.

I then proceeded to upgrade `pyqt5` with command `pip install --upgrade pyqt5`
This returned:
```
Defaulting to user installation because normal site-packages is not writeable
Requirement already up-to-date: pyqt5 in /usr/lib64/python3.8/site-packages (5.15.2)
Requirement already satisfied, skipping upgrade: PyQt5-sip<13,>=12.8 in /usr/lib64/python3.8/site-packages (from pyqt5) (12.8.1)
``` 
I executed `export QUTE_DARKMODE_VARIANT=qt_515_2` but qutebrowser version still shows _PyQt5.QtWebEngine: 5.15.1_

What happens when I install via git master? What affect does this have to the opensuse snapshot next time i upgrade? will it get overwritten?
